### DISCUSSION TITLE
Configure the Debug Service to use a CA signed certificate

### DISCUSSION THREAD

**Discussion by krka01** [codefori/vscode-ibmi#1396] [Q&A]
Hi,

How do you configure the Debug Service to use a CA signed certificate (requirement in our organisation) instead of the self signed certificate that is created when you use the guide? 
https://halcyon-tech.github.io/docs/#/pages/developing/debug/debug?id=configuring-the-debugger

The certificate I want to use is the same as the one we have generated CSR for in the DCM and that has been signed by Digicert.

Thanks in advance

**Comment by worksofliam** [comment]
Hi!

We have a PR open that will allow you to import a certificate. I suspect it will be out for 2.0.0.

**Reply by krka01** [reply]
Sounds good. Thanks

**Comment by krka01** [comment]
Hi,

I noticed version 2.0.0 is out now, did the import a certificate PR made it to 2.0.0? Is it this PR?: https://github.com/halcyon-tech/vscode-ibmi/commit/a0a63b7061d0f0ab632b2057ffc7c7d24cb5fe75  

Anyway, I can't find a way to import a CA signed server certificate for the Debug Service in version 2.0.0 either.
